modules/core/02-client/keeper/grpc_query_test.go

```diff
@@ -28,6 +28,12 @@ func (suite *KeeperTestSuite) TestQueryClientState() {
 		malleate func()
 		expPass  bool
 	}{
+		{"req is nil",
+			func() {
+				req = nil
+			},
+			false,
+		},
```

Comment:
this test case panics using the old old testing setup. Another reason to use the new one :slightly_smiling_face: